x-pack/plugins/endpoint/public/applications/endpoint/store/managing/middleware.test.ts

```diff
@@ -19,38 +20,14 @@ describe('endpoint list saga', () => {
   let store: Store<ManagementListState>;
   let getState: typeof store['getState'];
   let dispatch: Dispatch<AppAction>;
-  let history: History<never>;
 
+  const generator = new EndpointDocGenerator();
   // https://github.com/elastic/endpoint-app-team/issues/131
   const generateEndpoint = (): EndpointMetadata => {
-    return {
-      '@timestamp': new Date(1582231151055).toString(),
-      event: {
-        created: new Date(0),
-      },
-      endpoint: {
-        policy: {
-          id: '',
-        },
-      },
-      agent: {
-        version: '',
-        id: '',
-      },
-      host: {
-        id: '',
-        hostname: '',
-        ip: [''],
-        mac: [''],
-        os: {
-          name: '',
-          full: '',
-          version: '',
-          variant: '',
-        },
-      },
-    };
+    return generator.generateEndpointMetadata(new Date().getTime());
```

Comment:
You could just do `generator.generateEndpointMetadata()` since you added a default date.